src/main/java/com/ghilly/service/CountryServiceRest.java

```diff
@@ -0,0 +1,41 @@
+package com.ghilly.service;
+
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CountryServiceRest implements CountryService {
+
+    private static final Logger logger = LoggerFactory.getLogger(CountryServiceRest.class);
+
+
+    @Override
+    public String add(String countryName) {
+        logger.info("The country {} was added by service.", countryName);
+        return countryName;
+    }
+
+    @Override
+    public List<String> receiveList() {
```

Comment:
maybe would it be better come up with new more readable name?

Reply:
Ok.